netsim/ansible/templates/initial/junos.j2

```diff
@@ -21,7 +21,12 @@ system {
 interfaces {
 {# Handle MTU outside the main interface loop, since it needs to be applied only to the master interface #}
 {% for l in interfaces|default([]) %}
-{%   if l.mtu is defined %}
+{%   if l.mtu is defined and l._vlan_master|default(false) and l._vlan_mtu_override is defined %}
+{#   - i.e., can be 22 bytes more (depending on mode) #}
+  {{ l.junos_interface }} {
+    mtu {{ l.mtu + l._vlan_mtu_override }};
```

Comment:
```yaml
{%   if l.mtu is defined %}
  {{ l.junos_interface }} {
    mtu {{ l.mtu + (l._vlan_mtu_override|default(14) if l._vlan_master|default(false) else 14) }};
```

Reply:
Though I would suggest to put this logic of setting ```_vlan_mtu_override``` in the Python code. and in the template only do
```
mtu {{ l.mtu + l._vlan_mtu_override|default(14) }}
```